edx_name_affirmation/api.py

```diff
@@ -212,3 +214,38 @@ def update_is_verified_status(
         )
     )
     log.info(log_msg)
+
+
+def create_verified_name_config(user, use_verified_name_for_certs=None):
```

Comment:
With the django config model, the default is that a model can never be updated only created. Is this behavior what we want? Benefits are that we have a record of each config change, but that also means we will potentially have many configs for each user

Reply:
I think this is a config model because a user either wants to `user_verified_name_for_certs` set to `True`, or `False`, but not `null`. So the 2 states makes more sense to 3 states.